src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -102,9 +102,10 @@ public async Task<bool> PushMetadataAsync(CancellationToken cancellationToken)
             var local = new Local(logger, RepoRoot);
             IEnumerable<DependencyDetail> dependencies = await local.GetDependenciesAsync();
```

Comment:
@chcosta pinged me about an issue the nuget builds are having while trying to publish to BAR without having a Version.Details.xml. Originally, failing if there was none was by design since it is required to flow versions and define dependencies. For them, given that they don't depend on anything in BAR having a Version.Details.xml doesn't really make sense. So we could update this code to not throw if the file is not found or have them create an empty Version.Details.xml. Thoughts? FYI @mmitche

Reply:
Oh, I'm a bit behind on email. This is what I was referring here: https://github.com/dotnet/arcade/issues/2396